File: app/src/main/java/com/worldventures/dreamtrips/wallet/analytics/settings/DisableDefaultAction.java
package com.worldventures.dreamtrips.wallet.analytics.settings;

import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.core.service.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.Attribute;
import com.worldventures.dreamtrips.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:settings:disable default payment after",
                navigationState = true,
                trackers = AdobeTracker.TRACKER_KEY)
public class DisableDefaultAction extends WalletAnalyticsAction {

   @Attribute("disabledefault") final String disableDefault;

   public DisableDefaultAction(String disableDefaultTime) {
      this.disableDefault = disableDefaultTime;
   }
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/settings/DisableDefaultAction.kt
package com.worldventures.wallet.analytics.settings

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.Attribute
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:settings:disable default payment after", navigationState = true, trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class DisableDefaultAction(@field:Attribute("disabledefault") internal val disableDefault: String) : WalletAnalyticsAction()
